### PR TITLE
Fix #79 and remove some hard coded value in tests

### LINKED ISSUE #79: Normalization with log power in compute_pow_freq_bands
Hi ,

Just looking at the code, there might be an issue with the normalization with the combination `log=True`, `normalize=True`  in the function `compute_pow_freq_bands()` at https://github.com/mne-tools/mne-features/blob/399092ab015995afc66453ecc99fbe3e502f9a9a/mne_features/univariate.py#L722

```
    pow_freq_bands = np.empty((n_channels, n_freq_bands))
    for j in range(n_freq_bands):
        mask = np.logical_and(freqs >= fb[j, 0], freqs <= fb[j, 1])
        psd_band = psd[:, mask]
        pow_freq_bands[:, j] = np.sum(psd_band, axis=-1)

    if log:
        pow_freq_bands = np.log10(pow_freq_bands)

    if normalize:
        pow_freq_bands = np.divide(pow_freq_bands,
                                   np.sum(psd, axis=-1)[:, None])
```

If I'm not wrong the normalization should be done before apply the log:

```
  if normalize:
        pow_freq_bands = np.divide(pow_freq_bands,
                                   np.sum(psd, axis=-1)[:, None])

   if log:
        pow_freq_bands = np.log10(pow_freq_bands)

  
```

### PR DESCRIPTION
**Pull request by thomasfillon**
As discussed, here is my proposal for fixing #79.

I also change the retun value from log10 to dB (10 log10) and remove some hard coded power value in the corresponding test.

### CODE REVIEW

**Comment by thomasfillon** [comment]
> Thx a lot @thomasfillon

you're welcome @agramfort ;-)

> can I ask you to add a line in https://github.com/mne-tools/mne-features/blob/master/doc/whats_new.rst to document the fix ?

Done. Is it ok for you ? 

> also you should avoid opening PR from your master branch and use dedicated feature branches

Sorry for that. Should I delete the PR and move the commits to a new feature branch ?

File: doc/whats_new.rst [hunk 1]
```diff
@@ -43,13 +43,15 @@ Enhancements
...
 
 Bug
 ~~~
 
 - Fixed the behavior of the `edge` parameter in :func:`mne_features.univariate.compute_spect_edge_freq`. Valid values for this parameter are now `None` or a list of float between `0` and `1` (percentages). By `Jean-Baptiste SCHIRATTI`_ in `#52 <https://github.com/mne-tools/mne-features/pull/52>`_.
 - Fixed channel name replacement with overlapping channel names (e.g., Cz and FCz) which affected :func:`mne_features.extract_features` with `return_as_df=True` and when `ch_names` are provided. By `Hubert Banville`_ in `#63 <https://github.com/mne-tools/mne-features/pull/63>`_.
 - Fixed :class:`feature_extraction.FeatureFunctionTransformer`'s `check_input` method following sickit-learn 1.0 release: passing `reset=True`. By `Paul ROUJANSKY`_ in `#70 <https://github.com/mne-tools/mne-features/pull/70>`_.
+- Fixed the value of normalized `log-power` values in :func:`mne_features.univariate.compute_pow_freq_bands`. Normizatio by total power is done before taking the log of the power value.  By `Thomas Fillon`_ in `#80 <https://github.com/mne-tools/mne-features/pull/80>`_.
```
Comment (agramfort) [inline doc/whats_new.rst:54]:
```suggestion
- Fixed the value of normalized `log-power` values in :func:`mne_features.univariate.compute_pow_freq_bands`. Normalization by total power is now done before taking the log of the power value.  By `Thomas Fillon`_ in `#80 <https://github.com/mne-tools/mne-features/pull/80>`_.
```

**Review by agramfort — COMMENTED** [review]
see CI failures

https://github.com/mne-tools/mne-features/runs/6924264058?check_suite_focus=true

you need to run flake8 on your code to avoid issues

no need to change branch at this point. It will be for the next PR ;)

**Comment by agramfort** [comment]
thx @thomasfillon 🙏

**Pull request merged**